packages/nextjs/pages/builders/0x3b529ac4F8dc0C40Fec0f1192029755275A0c3Cd.tsx

```diff
@@ -0,0 +1,37 @@
+import Image from "next/image";
+
+export default function KavirajProfilePage() {
```

Comment:
Could you use `const` here and export it at the bottom of the file? Just to be consistent within the project.